RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -116,6 +107,31 @@
 ))
 highBetaStar_2018.toModify(detachedTripletStepHitTriplets,CAThetaCut = 0.002,CAPhiCut = 0.1,CAHardPtCut = 0)
 
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+_fastSim_detachedTripletStepSeeds = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    trackingRegions = "detachedTripletStepTrackingRegions",
+    hitMasks = cms.InputTag("detachedTripletStepMasks"),
+    seedFinderSelector = dict( pixelTripletGeneratorFactory = _hitSetProducerToFactoryPSet(detachedTripletStepHitTriplets),
+                               layerList = detachedTripletStepSeedLayers.layerList.value())
+)
+#new for phase1
+trackingPhase1.toModify(_fastSim_detachedTripletStepSeeds, seedFinderSelector = dict(
+        pixelTripletGeneratorFactory = None,
+        CAHitTripletGeneratorFactory = _hitSetProducerToFactoryPSet(detachedTripletStepHitTriplets).clone(SeedComparitorPSet = dict(ComponentName = "none")),
```

Comment:
Looking the test failures, `detachedTripletStepHitTriplets` does not have `SeedComparitorPSet` parameter, so you should drop the customizations inside the `clone()` call.

Reply:
Right. I will remove it.